packages/playground/data-liberation/src/import/WP_Topological_Sorter.php

```diff
@@ -0,0 +1,461 @@
+<?php
+
+/**
+ * The topological sorter class.
+ *
+ * We create an in-memory index that contains offsets and lengths of items in the WXR.
+ * The indexer will also topologically sort posts so that the order we iterate over posts
+ * ensures we always get parents before their children.
+ */
+class WP_Topological_Sorter {
+
+	/**
+	 * The base name of the table.
+	 */
+	const TABLE_NAME = 'data_liberation_index';
```

Comment:
This is called index, but it's not used in the `index_next_entities()` stage, only in the topological sort stage. How about `data_liberation_topological_index` or something to that effect?